app/controllers/api/v1/lineups_controller.rb

```diff
@@ -0,0 +1,59 @@
+class API::V1::LineupsController < ApplicationController
+  include ViewPermissionConcern
+
+  def index
+    render :json => Lineup.all
```

Comment:
Could use some excepts, like update_at, created_at and other unneeded information